Description: fix xlocale.h include for glibc 2.26+ compat
 This patch fixes compatibility with glibc 2.26+ due to xlocale.h removal.
Origin: vendor, https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=opencascade7
Author: Grey Christoforo
Bug: https://tracker.dev.opencascade.org/view.php?id=28971.
===================================================================

--- src/Standard/Standard_CLocaleSentry.hxx
+++ src/Standard/Standard_CLocaleSentry.hxx
@@ -37,7 +37,7 @@
 
 #if defined(HAVE_XLOCALE_H) && !(defined(__GLIBC__) && (__GLIBC__ == 2 && __GLIBC_MINOR__ <= 24))
   // xlocale.h is actually a non-standard header file; glibc 2.26 has removed it altogether (all definition comes from locale.h)
-  #include <xlocale.h>
+  #include <X11/Xlocale.h>
 #endif
 
 #if !defined(__ANDROID__)

